jepsen/project.clj

```diff
@@ -9,7 +9,7 @@
                  [org.clojure/tools.cli "0.3.5"]
                  [clj-time "0.11.0"]
                  [jepsen.txn "0.1.0"]
-                 [knossos "0.3.3-yb" :exclusions [org.slf4j/slf4j-log4j12]]
+                 [knossos "0.3.3-yb-SNAPSHOT" :exclusions [org.slf4j/slf4j-log4j12]]
```

Comment:
Would probably be better if we don't depend on a SNAPSHOT build. Is it possible to bump up and push the next version, so we don't acccidentally break this if we were to push code to knossos?